free-1.md

```diff
@@ -10,34 +10,84 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Trying to understand someone else's code&mdash;especially at a large scale&mdash;can be an intimidating, difficult, and overwhelming task,
+even if the person trying to digest the code is an experienced programmer:
+having to read and distinguish keywords in raw source code can be tiring, even if the code is well-commented.
+Documentation can help this process, but words on a page aren't always the best way to express an idea.
+To address this problem, I propose designing a language that translates code into flowcharts,
+which would offer a more visually-appealing representation of code.
+Not only would this solution help anyone (i.e., programmers and non-programmers alike) in understanding code,
+but it could also reduce the effort necessary to produce 'good' documentation,
+and (perhaps optimistically) force programmers to write more high level and well-commented/documented code.
+One really cool potential result would be that people could use skeleton code in the language in which they intend to work
+(and with which, therefore, they _are probably_, or at least _probably should_, be familiar)
+to generate high level flowcharts that express the intended algorithm.
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+1. The programmer will likely not want every implementation detail to leak into the flowcharts,
+so there should be some way to indicate what _should_ appear on the flowcharts.
+
+2. There should be as few barriers as possible to using this tool,
+so I think allowing users to simply add code or annotations to the working code would be ideal.
+
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I really like the idea of making algorithms represented in code more accessible.
+I myself tend to favor and think in visualizations, and I think it would be a lot easier to explain code to someone
+if I could easily transform my code into a flowchart. During my internship over the summer,
+I wrote a document that details the design of my project, and in that document,
+I created a flowchart to represent how I decided to handle one of the more complex parts of the project
+because a flowchart seemed fitting for the task. I also think documentation is _super_ important,
+so it's very appealing to me to create tools that ameliorate the documentation process
+and compel others to put more effort toward it.
+
```

Comment:
Right! The flowcharts would just be another tool for describing workflow&mdash;to complement syntax and usage documentation.